app/containers/DocumentList/DocumentList.jsx

```diff
@@ -193,25 +209,17 @@ class DocumentList extends React.Component {
     const documentsAreLoading = !data || data.isLoading
 
     if (documentsAreLoading) {
-      // If there is a custom handler for the loading state, we'll call it.
-      if (typeof onLoading === 'function') {
-        return onLoading()
-      }
+      return typeof onLoading === 'function' ? onLoading() : <SpinningWheel />
+    }
 
-      // If not, we render the SpinningWheel component.
-      return <SpinningWheel />
+    if (data.error) {
+      return typeof onError === 'function' ? onError({error: data.error}) : null
     }
 
     const {metadata, results} = data
 
-    // If there are no results to display, we call the `onEmptyList` callback
```

Comment:
Here I took the liberty to clear out some comments which were telling me the same thing as the code, if that's okay :slightly_smiling_face: